packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -218,12 +218,29 @@ export class EthereumConnection implements BcpConnection {
   }
 
   public async getAllTickers(): Promise<ReadonlyArray<BcpTicker>> {
+    const erc20s = await Promise.all(
+      [...this.erc20Tokens.entries()].map(
+        async ([ticker, contract]): Promise<BcpTicker> => {
+          if (ticker !== (await contract.symbol())) {
+            throw new Error("Inconsistent ticker configured and in contract");
+          }
+
+          return {
+            tokenTicker: (await contract.symbol()) as TokenTicker,
```

Comment:
Is this an expensive query? No need to do it twice if so.

Reply:
It is explicitly documented to be cheap:

```ts
    /**
     * Returns symbol value from options or from chain.
     *
     * On-chain values will be cached internally, i.e. it is cheap to use this getter
     * as long as the class instance is long living.
     */
    name(): Promise<string>;
    /**
     * Returns symbol value from options or from chain.
     *
     * On-chain values will be cached internally, i.e. it is cheap to use this getter
     * as long as the class instance is long living.
     */
    symbol(): Promise<string>;
    /**
     * Returns decimals value from options or from chain.
     *
     * On-chain values will be cached internally, i.e. it is cheap to use this getter
     * as long as the class instance is long living.
     */
    decimals(): Promise<number>;
```